cmd/graphsplit/main.go

```diff
@@ -66,6 +67,11 @@ var chunkCmd = &cli.Command{
 			Value: true,
 			Usage: "create a mainfest.csv in car-dir to save mapping of data-cids and slice names",
 		},
+		&cli.BoolFlag{
+			Name:  "calc-commp",
+			Value: false,
+			Usage: "calculate pieceCID and pieceSize",
+		},
```

Comment:
Usage is recommended to be changed to "create a mainfest.csv in car-dir to save mapping of data-cids, slice names, piece-CID and piece-size" for ease of understanding.